### ISSUE TITLE
Request: Download from paid emumovies account

### ISSUE THREAD

**Issue by hanthecat** [sselph/scraper#105]
As an option allow downloading from an emumovies account.
This will resolve any issues people may have with lower quality images.

**Comment by robertybob** [comment]
I have an account there but the problem I found that was the box-art images were mainly fake/recreated- well, at least for the Nintendo systems. 

I could see this being beneficial for some, but if thegamesdb.net has low quality images then people need to upload better ones there so everyone can benefit :)

**Comment by hanthecat** [comment]
I have been trying to add some to thegamedb -- The site is just so slow. D=

**Comment by robertybob** [comment]
Yeah that's fair enough, thats down to its popularity more than anything. 
FYI I'm a mod there so anything you submit should be actioned within a day or two :)

**Comment by paradadf** [comment]
I recommend you to take a look at [screenscraper.fr](screenscraper.fr). In my opinion, that is a database worth considering to offer support for. It is still pretty new, but the quality of the media is superior to tgdb. And the developers are very responsive over IRC.

**Comment by stevetb** [comment]
@sselph I really would tend to agree with the request from @hanthecat .  While TheGamesDB is a decent database, it is very much a grassroots effort being done by a small few.  I do not consider TheGamesDB to be stable or fast.  We have seen the server go down for weeks and while I find it fast enough through your scraper, the website is slow.  With EmuMovies, moving to that we get access to a DB which is not going anywhere.  It's stable with very well done images.  I personally do not care if the images are faked, they look great.  With TheGamesDB, we do get direct access to their DB through their API.  EmuMovies may have a much more locked down ecosystem.

Let me know if you are interested in this @sselph and I will do what I can to help you map across to the EmuMovies ID's.  Sounds like a good winter project :)

**Comment by paradadf** [comment]
In case you are interested, and even if the page is not translated yet, here is information about the API from ScreenScraper.

http://www.screenscraper.fr/webapi.php?alpha=0&numpage=0

There is another API with info matching 100% with the TGDB API, so it shouldn't be difficult to modify the scraper, but for that you should ask the developer directly. And you would need a DevID ;)
The best is to go into IRC and chat with MarbleMad.
What I like about ScreenScraper is that it is CC and of couse, free. As I said before, the quality is amazing!

**Comment by hanthecat** [comment]
I am going to have to check this out! Parlez vous? ... wish I did. :P

**Comment by sselph** [comment]
@hanthecat Do you have a link to any documentation for emumovies? I can't seem to find much on their site. They seem to have their own sync tool and the forum section for it implies there is an API but I don't see anything documented. I have #102 where I was asked to add a new image source but that requires some work to change how I'm handling images. Once I figure that out I can look to expand this but will need to see if this is feasible.

@paradadf Thanks for pointing out a new data source. I like that they have the option to query based on hash but I don't like how it appears they have done it but I get why. They have just hashed the raw files in different formats to try to get a comprehensive list so for some systems this may work but for others like NES where there are headers involved it might not. I can try reaching out to their devs to see if I'm just misunderstanding what I think I'm seeing, but if we can provide a good hash and get the data this means that we don't have to migrate any of the mapping code and things will just work. I have to warn you, I haven't been doing much development in my free time so may take a little while.

**Comment by stevetb** [comment]
@sselph I think the Emu Movies developers would really need to be consulted about how best to integrate.  Looking through their site I did not find any API explanations or integration assistance.  The best way might be to reach out to them in their forums and ask for assistance in integrating.  Let me know if you need help and I can help push this through.

**Comment by pgbconfirmit** [comment]
I would also love to see support added for EmuMovies.
